### ISSUE TITLE
Divide by zero when experimental test runner stats is enabled

### ISSUE THREAD

**Issue by xfthhxk** [mhjort/trombi#97]
Hi @mhjort! Thanks for creating this library.  I was taking trombi for a spin and got this exception when trying out the `:experimental-test-runner-stats?` option.  
```
Exception in thread "async-thread-macro-38" java.lang.ArithmeticException: Divide by zero
	at clojure.lang.Numbers.divide(Numbers.java:190)
	at clojure.lang.Numbers.divide(Numbers.java:3895)
	at trombi.stats$calculate_stats.invokeStatic(stats.clj:8)
	at trombi.stats$calculate_stats.invoke(stats.clj:6)
	at trombi.stats$start_stats_gathering$fn__22350.invoke(stats.clj:26)
	at trombi.pipeline$simulation_runner$fn__22373.invoke(pipeline.clj:50)
	at clojure.core.async$thread_call$fn__18425.invoke(async.clj:486)
	at clojure.lang.AFn.run(AFn.java:22)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
```

**Comment by mhjort** [comment]
Hi @xfthhxk ! Thanks for reporting. I just fixed the issue and released version `1.0.1` with the fix.

**Comment by xfthhxk** [comment]
Thanks! @mhjort I can confirm the issues is fixed with the `1.0.1` release.
